packages/api/src/index.ts

```diff
@@ -106,6 +106,7 @@ export type AppState = {
     };
   };
   registeredEmps: Set<string>;
+  registeredEmpsMetadata: Map<string, { blockNumber: number }>;
```

Comment:
@daywiss Is it ok to use the `Map` instead of the sdk's `JsMap`?

Reply:
yea its ok for now